app/renuo_upload_signing.rb

```diff
@@ -14,13 +14,6 @@ class RenuoUploadSigning < Sinatra::Base
     end
   end
 
-  get '/list_files' do
-    api_key = settings.authentication.api_key_or_nil(params[:api_key])
-    response_wrapper(api_key, 200) do
```

Comment:
for me it looks like a mistake: 
```rb
    private_key_valid = settings.authentication.private_api_key_valid?(api_key, params[:private_api_key])
```
should be also checked. then it would be ok, right?

Reply:
Yes, you are right. I'll adapt it.